blocks/breadcrumbs/breadcrumbs.css

```diff
@@ -0,0 +1,102 @@
+.breadcrumbs {
+  list-style-type: none;
+}
+
+.breadcrumbs-hidden {
+  visibility: hidden;
+}
+
+.breadcrumbs ol {
+  display: flex;
+  flex-wrap: nowrap;
+  font-family: var(--body-font-family);
+  font-size: var(--body-font-size-xxs);
+  color: var(--breadcrumbs-text-color);
+  line-height: 1rem;
+  letter-spacing: 0;
+  -webkit-font-smoothing: antialiased;
+  list-style-type: none;
+  padding-inline-start: 0;
+  margin: 0;
+  padding-bottom: 56px;
+}
+
+.breadcrumbs ol li {
+  padding: 0;
+  display: none;
+}
+
+.breadcrumbs li::after {
+  background: url('../../icons/chevron-right.svg');
```

Comment:
@rhudea , the svgs need to be added to the PR

Reply:
> @rhudea , the svgs need to be added to the PR

Those are in sharepoint